### DISCUSSION TITLE
How to hide the left sidebar?

### DISCUSSION THREAD

**Discussion by olegopro** [Kong/insomnia#6365] [Help]
How to hide the left sidebar in which there are square icons of personal projects? I couldn't find this feature in the settings.

I really love Insomnia for its minimalistic interface, but it's becoming more and more like an ugly Postman

**Comment by subnetmarco** [comment]
This is going to be possible next week with 9.0: https://github.com/Kong/insomnia/discussions/7069#discussioncomment-9144919
